twingate/internal/utils/utils.go

```diff
@@ -22,3 +23,32 @@ func Filter[T any](items []T, ok func(item T) bool) []T {
 
 	return result
 }
+
+// Contains - checks if element exists in the slice.
+func Contains[T comparable](items []T, element T) bool {
+	lookup := MakeLookupMap[T](items)
+
+	return lookup[element]
+}
+
+// MapKeys - collects map keys to slice.
+func MapKeys[T comparable](lookup map[T]bool) []T {
```

Comment:
There's something fishy about these 2 methods - `MapKeys`, `MakeLookupMap` - converting a list of values to a map of `value -> true` and back.
Doesn't Golang has a `set` ?

Reply:
Bah thats the Golang way it seems...

Reply:
yeah, golang doesn't has `set`